### PR TITLE
Adding hidePlaceholderMenu parameter

### PR DESCRIPTION
**Pull request by ahmad-shahid**

### CODE REVIEW

File: gocwebtemplate-core/gocwebtemplate-core-base/src/main/java/goc/webtemplate/component/AbstractCoreBean.java [hunk 1]
```diff
@@ -1452,6 +1453,24 @@ public void setCustomFooterSections(ArrayList<FooterSection> value) {
         this.customFooterSections = value;
     }
     
+    /**
+     * Returns whether the placeholder menu should be hidden while a custom menu is being loaded
+     *
+     * can be overriden programatically.
+     */
+    public boolean getHidePlaceholderMenu() {
+        return this.hidePlaceholderMenu;
```
Comment (plup2) [inline gocwebtemplate-core/gocwebtemplate-core-base/src/main/java/goc/webtemplate/component/AbstractCoreBean.java:1462]: Abosutely need a `this.initializeOnce();` before returning the value (to give a chance to overriders to set the value)

File: gocwebtemplate-core/gocwebtemplate-core-base/src/main/java/goc/webtemplate/component/AbstractCoreBean.java [hunk 2]
```diff
@@ -1797,7 +1816,8 @@ public String getRenderTop() {
                     true, //siteMenu
                     this.getHasLeftMenuSections(), //topSecMenu, true if there is at least one left menu section defined
                     this.customSearch != null? Arrays.asList(this.customSearch): null,      
-                    this.gcToolsModal
+                    this.gcToolsModal,
+                    this.hidePlaceholderMenu
```
Comment (plup2) [inline gocwebtemplate-core/gocwebtemplate-core-base/src/main/java/goc/webtemplate/component/AbstractCoreBean.java:1820]: Unit test :)

File: gocwebtemplate-sample-jsp/src/main/resources/goc/webtemplate/global/config/cdn.properties [hunk 3]
```diff
@@ -29,7 +29,7 @@ cdn_localdev_localpath=/GoCWebTemplateSampleJSP/loadtesting/app/cls/WET/{0}/{1}/
 #(blank values for theme, subtheme and version will use defaults from internal environment configuration)
 wettemplate_theme=
 wettemplate_subtheme=
-wettemplate_version=v4_0_45
+wettemplate_version=v4_0_46
```
Comment (plup2) [inline gocwebtemplate-sample-jsp/src/main/resources/goc/webtemplate/global/config/cdn.properties:32]: Try with blank to see if JSP sample still works fine, if it does let's remove it.

**Review by plup2 — CHANGES_REQUESTED** [review]

File: gocwebtemplate-core/gocwebtemplate-core-base/src/main/java/goc/webtemplate/component/AbstractCoreBean.java (hunk 1 above)
Comment (ahmad-shahid) [reply gocwebtemplate-core/gocwebtemplate-core-base/src/main/java/goc/webtemplate/component/AbstractCoreBean.java:1463]: done

File: gocwebtemplate-core/gocwebtemplate-core-base/src/main/java/goc/webtemplate/component/AbstractCoreBean.java (hunk 2 above)
Comment (ahmad-shahid) [reply gocwebtemplate-core/gocwebtemplate-core-base/src/main/java/goc/webtemplate/component/AbstractCoreBean.java:1821]: done

File: gocwebtemplate-sample-jsp/src/main/resources/goc/webtemplate/global/config/cdn.properties (hunk 3 above)
Comment (ahmad-shahid) [reply gocwebtemplate-sample-jsp/src/main/resources/goc/webtemplate/global/config/cdn.properties]: Tested with blank and it works perfectly.

**Review by plup2 — APPROVED** [review]

**Comment by plup2** [comment]
@ahmad-shahid Somebody forgot to sign their commit :)

**Comment by ahmad-shahid** [comment]
> @ahmad-shahid Somebody forgot to sign their commit :)

lol ...just did now :)

**Pull request merged**